OpenCeedlingFile.py

```diff
@@ -1,71 +1,84 @@
-import sublime
+from __future__ import print_function
+import os
+import re
+
 import sublime_plugin
-import re, inspect, os
+
 
 class OpenCeedlingFileCommand(sublime_plugin.WindowCommand):
+    def run(self, option):
+        if not self.window.active_view():
+            return
+
+        self.views = []
+        window = self.window
+        current_file_path = self.window.active_view().file_name()
+
+        if option == 'config':
+            config_matcher = re.compile(r"(\\\\)?(project|test)\.yml$")
+            self.open_project_file(config_matcher, self.window)
+
+        elif re.search(r"\w+\.[ch]\w*$", current_file_path):
+
+            current_file = re.search(r"([\w\.]+)$", current_file_path).group(1)
+            base_name = re.search(r"(\w+)\.(\w+)$", current_file).group(1)
+            base_name = re.sub('^test_', '', base_name)

```

Comment:
There are better ways to parse and manipulate path strings using the `os.path` module.